solution/backend/regulations/templates/regulations/partials/node_types/Paragraph.html

```diff
@@ -16,11 +16,12 @@
             hover
             click
         >
-            <copy-citation
-                slot="tooltip-content"
-                hash="{{node.label | join:'-'}}"
-                formatted-citation="{{formatted_citation}}"
-            ></copy-citation>
+            <template #tooltip-content>
+                <copy-citation
+                    hash="{{node.label | join:'-'}}"
+                    formatted-citation="{{formatted_citation}}"
+                ></copy-citation>
+            </template>
```

Comment:
Named slots must be used on a <template> element. See https://vuejs.org/guide/components/slots.html#named-slots